### PR TITLE
Fixxer

### PR DESCRIPTION
**Pull request by akrherz**

### CODE REVIEW

**Comment by akrherz** [comment]
ping @guusdk :)

**Comment by guusdk** [comment]
Do you want to update the release date, given my slacking off?

File: src/web/export-file.jsp [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<%@ page contentType="text/html; charset=UTF-8" %>
```
Comment (guusdk) [inline src/web/export-file.jsp]: I think I already addressed this in https://github.com/igniterealtime/openfire-userImportExport-plugin/commit/c8e2ebbc0813cac2a50c998930868a293399010f

**Comment by akrherz** [comment]
I rebased it and too lazy to fix the release date.

**Comment by guusdk** [comment]
My only concern is the removal of the `application/x-download` header. I've got no idea if that has undesired side-effects.

**Comment by akrherz** [comment]
Isn't already set by line 5?

**Comment by guusdk** [comment]
*mumble*mumble*coffee

**Pull request merged**